Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "JustRideSDK",
```

Comment:
We'll have to leave this with a big 'R' so existing SDK Integrators who switch over to getting the Justride SDK via Swift Package Manager don't encounter compilation errors in their code.